openassessment/xblock/static/js/spec/oa_server.js

```diff
@@ -187,7 +187,7 @@ describe("OpenAssessment.Server", function() {
         expect($.ajax).toHaveBeenCalledWith({
             url: "/save_files_descriptions",
             type: "POST",
-            data: JSON.stringify({descriptions: ['test1', 'test2']}),
+            data: JSON.stringify({fileMetadata: ['test1', 'test2']}),
```

Comment:
Can we add a test that indicates that file names are also submitted in this POST request?